### DISCUSSION TITLE
[Javascript] Custom scripts for Thunderbird

### DISCUSSION THREAD

**Discussion by Aris-t2** [Aris-t2/CustomCSSforTb#9] [General]
Some scripts offered for Firefox work on Thunderbird too.

Tested with method 2 and Thunderbird 68.1 on Windows 10.

[[HowTo/Setup] - https://github.com/Aris-t2/CustomJSforFx](https://github.com/Aris-t2/CustomJSforFx)

[[JavaScript][Thunderbird] General JavaScript talk and custom userChrome.js scripts](https://github.com/Aris-t2/CustomJSforFx/issues/30)

[Scripts](https://github.com/Aris-t2/CustomJSforFx/tree/master/scripts_thunderbird)

**Comment by Belou-43** [comment]
Hello,
I'm french, and my English is poor !!!
I had a "userChrome.css" before, to color unread mails in red.
Since Thunderbird is 68 version, it doesn'work anymore
I tried to insert the code in a new CSS script in your "custom_css_for_tb_v1.0.1", but it doesn' work
Here is my work:
In userChrome.css: (extract)


```css
/**************************************************************************************************/

/* Affichage des courriers non lus en ROUGE *****************************************************/
@import "./css/general/non-lus_rouges.css"; /**/

/* MAIN CONFIG FILE - controls variables ********************************************************/
/* @import "./config/config.css"; /**/

/* TABS *****************************************************************************************/

/* Tab appearance - [only use one at a time] ****************************************************/
@import "./css/tabs/classic_squared_tabs.css"; /**/
/* @import "./css/tabs/classic_squared_tabs_aero.css"; /**/
/* @import "./css/tabs/classic_squared_tabs_glass1.css"; /**/
/* @import "./css/tabs/classic_squared_tabs_glass2.css"; /**/
```


File : "non-lus_rouges.css" (In ./css/general/):

```css
/* Messagepane text */
treechildren:-moz-tree-cell-text(unread) {
font-family : Arial !important;
font-weight: Bold !important;
color: #FF0000 !important }
```


COULD YOU EXPLAIN ME WHERE IS MY ERROR ?????? (And help me to have a good "non-lus_rouges.css"
Thanks

**Comment by Belou-43** [comment]
My 2 files !!!
Thanks
[non-lus_rouges.zip](https://github.com/Aris-t2/CustomCSSforTb/files/3662520/non-lus_rouges.zip)

**Comment by Aris-t2** [comment]
It has nothing to do with CSS or JS. Mozilla introduced a config settings in Thunderbird 68, that blocks custom stuff by default. Open your Thunderbird settings, enter config area and set `toolkit.legacyUserProfileCustomizations.stylesheets` to `true`.

This is also mentioned here: https://github.com/Aris-t2/CustomCSSforTb/blob/master/stylesheets/userChrome.css#L13

**Comment by Belou-43** [comment]
I have already "toolkit.legacyUserProfileCustomizations.stylesheets" TO TRUE
My old "userChrome.css" was working fine
Now, it does'nt
Your "userChrome.css" and CSS "extentions" work fine, but not mine !!!!!!
I think my "extension" (non-lus_rouges.css) has a bad syntax ???
Thanks

**Comment by Aris-t2** [comment]
Your syntax is correct, I just think there is no `treechildren:-moz-tree-cell-text(unread)` rule anymore.

**Comment by Belou-43** [comment]
Thanks for all
PS: If you think about a solution......

**Comment by krystian3w** [comment]
@Aris-t2  use "<code>```css</code>" on this post: https://github.com/Aris-t2/CustomCSSforTb/issues/2#issuecomment-535901008
